project/src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,18 @@
+import {Navigate} from 'react-router-dom';
+import {AppRoute, AuthorizationStatus} from '../../const';
+
+type PrivateRouteProps = {
+  authorizationStatus: AuthorizationStatus;
+  children: JSX.Element;
+}
+
+function PrivateRoute(props: PrivateRouteProps){
```

Comment:
Лучше сразу вытаскивать нужные пропсы, если их очень много обычно располагают в столбик
```suggestion
function PrivateRoute({ authorizationStatus, children }: PrivateRouteProps){
```